docker/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM mambaorg/micromamba:2.4.0-ubuntu24.04
+FROM ghcr.io/prefix-dev/pixi:0.62.2-noble
```

Comment:
This is still ubuntu24.04 under the hood. They're just using the code name instead of the version number.